Source/m_menu.c

```diff
@@ -5198,10 +5198,11 @@ boolean M_Responder (event_t* ev)
 
         if (M_InputActivated(input_demo_fforward))
         {
-          if (demoplayback && singledemo && !demo_skipping)
+          if (demoplayback && singledemo && !demo_skipping && !fastdemo)
           {
-            fastdemo = !fastdemo;
-            I_SetFastdemoTimer();
+            static boolean on = false;
+            on = !on;
```

Comment:
This is confusing.

Reply:
I specifically mean the `on = !on` line. Could you rename the variable to something different, e.g. `fast`?